frontend/apps/crates/entry/home/src/home/dom/mod.rs

```diff
@@ -18,7 +18,9 @@ mod search_section;
 
 pub fn render(state: Rc<State>, auto_search: bool) -> Dom {
     html!("home-full", {
-        .child(page_header::dom::render(Rc::new(page_header::state::State::new()), None, Some(PageLinks::Home)))
+        .child_signal(state.mode.signal_ref(|mode| {
+            Some(page_header::dom::render(Rc::new(page_header::state::State::new()), None, Some(PageLinks::from(mode))))
```

Comment:
This could also be 

```suggestion
            Some(page_header::dom::render(Rc::new(page_header::state::State::new()), None, Some(mode.into())))
```